src/components/account/FindPasswordForm.tsx

```diff
@@ -1,34 +1,73 @@
 import styled from '@emotion/styled';
-import { useFormContext } from 'react-hook-form';
+import { useForm } from 'react-hook-form';
 import type { RegisterForm } from 'types/register';
+import { Button } from 'components/common';
 import { FormInput } from 'components/form';
 import { ERROR_MESSAGE, VALID_VALUE } from 'constants/validation';
 
-export const FindPasswordForm = () => {
+export const FindPasswordForm = ({ onSubmit }: { onSubmit: () => void }) => {
```

Comment:
p3;
FindPasswordForm props 인터페이스 따로 선언해서 적용 부탁드릴게요!
```suggestion
interface FindPasswordFormProps {
  onSubmit: () => void; 
}

export const FindPasswordForm = ({ onSubmit }: FindPasswordFormProps) => {
```

Reply:
반영했습니다~! [845f012](https://github.com/a-daily-diary/ADD.FE/pull/191/commits/845f0129528822047b6bbb3dea71eee67de768e6)